lumigator/python/mzai/backend/backend/settings.py

```diff
@@ -99,11 +99,20 @@ def RAY_WORKER_GPUS(self) -> float:  # noqa: N802
     def RAY_WORKER_GPUS_FRACTION(self) -> float:  # noqa: N802
         return float(os.environ.get(self.RAY_WORKER_GPUS_FRACTION_ENV_VAR, 1.0))
 
+    # URL for the Ray Dashboard
     @computed_field
     @property
     def RAY_DASHBOARD_URL(self) -> str:  # noqa: N802
         return f"http://{self.RAY_HEAD_NODE_HOST}:{self.RAY_DASHBOARD_PORT}"
 
+    # URL for Ray jobs API
+    @computed_field
+    @property
+    def ray_jobs(self) -> str:  # noqa: N802
```

Comment:
❓ Should we match the UPPERCASING we have with other methods we have in here? (e.g. `RAY_JOBS` to match `RAY_DASHBOARD_URL`, `SQLALCHEMY_DATABASE_URL` etc.).

❓ What do you think about `RAY_JOBS_URL` (vs. `RAY_JOBS`) to align with the other methods that are returning a 'URL' (as a URL or as a string).

Reply:
1) Could be. I didn't check if properties were all also in uppercase. I'll check.

1) Fine for me 👍